tests/test.py

```diff
@@ -1636,6 +1689,9 @@ def testClosingConnectionsWithDB(self):
 class TestKeyNonExistence(BaseTestCase):
     def setUp(self):
         self.checkSkip()
+        # FIXME(ilammy, 2020-03-20): implement key removal for v2
```

Comment:
add please "TODO" abbr, to collect it by IDE (jetbrains' ide have tab TODO which aggregate all todos from all files. maybe some other tools have same feature like grep by "todo" word)

Reply:
PyCharm shows FIXMEs in the TODO list too.

My idea is that TODO is something that you have a moral right to ignore, like a use case that is not useful right now but might become important later. Or some really-really rare edge case. So if you see a TODO in code you don't go all “why didn't you implement it then?”

While FIXME is something that's actually broken and, well, needs a fix. It's probably not *really* important bug that may be ignore right now. But you *should* become angry at the person who left a fixme instead of fixing their stuff.

Anyhow, I don't care particularly so I can rename it if you insist.

Reply:
if pycharm show fixmes too than ok. plus I thought that we can "todo" keyword to description like `# FIXME(ilammy, 2020-03-20): todo: ` if it wasn't match by IDE.